Doc/library/functions.rst

```diff
@@ -1729,15 +1731,15 @@ are always available.  They are listed here in alphabetical order.
 
 
 .. _func-range:
-.. class:: range(stop)
-           range(start, stop, step=1)
+.. class:: range(stop, /)
+           range(start, stop, step=1, /)
```

Comment:
See https://github.com/python/cpython/pull/125945 for range

Reply:
That PR, submitted last Octover, proposed these same changes.  There was some discussion about adding the `/`s, perhaps in part because only in one place, but there are other approvals.  It should be merged before this one so that the OP, a new contributor, gets proper credit.

Reply:
Merged